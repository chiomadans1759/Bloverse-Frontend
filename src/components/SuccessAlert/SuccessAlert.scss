#success-alert {
  .success {
    background: #02DE84;
    border-radius: 0;
    color: #ffffff;
    font-weight: 600;
    border: 0.063rem solid rgba(0, 0, 0, 0.15);
    padding: 1.5rem;
  }
  .text {
    font-size: 0.875rem;
  }
}