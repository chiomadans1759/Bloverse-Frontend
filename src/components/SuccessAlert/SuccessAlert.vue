<template>
  <main id="success-alert">
    <div class="alert alert-success success" role="alert">
     <h4 v-if="heading" class="alert-heading" >{{heading}}</h4>
     <p class="text"><slot></slot></p>
    </div>
  </main>
</template>

<script>
export default {
  name:'Error',
  props: {
    heading: {
      type: String,
      required: false,
      default: ""
    }
  } 
}
</script>

<style lang="scss" scoped>
  @import './SuccessAlert';
</style>

