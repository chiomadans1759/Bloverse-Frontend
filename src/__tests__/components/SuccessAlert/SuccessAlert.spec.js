import { mount } from '@vue/test-utils';
import SuccessAlert from '@/components/SuccessAlert/SuccessAlert.vue';

describe('SuccessAlert Component', () => {
  it('should render correctly', () => {
    const wrapper = mount(SuccessAlert);

    expect(wrapper.element).toMatchSnapshot();
  });
  it('should render title', () => {
    const wrapper = mount(SuccessAlert, {
      propsData: {
        heading: 'alert-heading'
      }
    })
    expect(wrapper.props().heading).toBe('alert-heading');
  })
});
