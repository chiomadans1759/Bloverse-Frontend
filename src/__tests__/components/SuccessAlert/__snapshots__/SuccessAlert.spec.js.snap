// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessAlert Component should render correctly 1`] = `
<main
  id="success-alert"
>
  <div
    class="alert alert-success success"
    role="alert"
  >
    <!---->
     
    <p
      class="text"
    />
  </div>
</main>
`;
